protocol/contracts/libraries/Silo/LibSilo.sol

```diff
@@ -491,8 +491,8 @@ library LibSilo {
     function getSopToken() internal view returns (IERC20) {
```

Comment:
this function should be updated with gen flood. `getSopToken()` is used in the context of invariable.getTokensOfInterest. Defer to you on how you want to implment it, but the goal is to make `getTokensOfInterest` include all the sop tokens.

Reply:
cc @funderbrker  for visability

Reply:
i will need to patch. could be a bit hairy. @pizzaman1337 lmk when is a good time to jump in on this branch

Reply:
Here's my shot, lmk: https://github.com/BeanstalkFarms/Beanstalk/pull/870/commits/4db1db8fe9726ed04a554160d62a276912c116ac

Reply:
commit looks gud